skytap/vm.go

```diff
@@ -458,7 +434,7 @@ func (s *VMsServiceClient) changeRunstate(ctx context.Context, environmentID str
 	}
 
 	var updatedVM VM
-	_, err = s.client.do(ctx, requestCreate, &updatedVM)
+	_, err = s.client.do(ctx, requestCreate, &updatedVM, vmRunStateNotBusy(environmentID, id), opts)
```

Comment:
This needs to check that the environment is not busy. When a VM is busy, its environment is also busy, so attempting to set runstate on other VMs will fail with 423 Locked.

Reply:
Even though the logic as it currently is will work - I will make this change for completeness. In case you were not aware: a 423 error will trigger a retry. Implementing this change will clean up the logs by removing the 423 errors.